doc/release-notes-32604.md

```diff
@@ -0,0 +1,8 @@
+Logging
+-------
+Unconditional logging to disk is now rate limited via `LogPrintf`, `LogInfo`,
+`LogWarning, `LogError`, and the corresponding `LogPrintLevel` calls by giving
+each source location a quota of 1MiB per hour. (#32604)
```

Comment:
Maybe worth adding that "All logs will be prefixed with [*] if there is at least one source location that is currently being suppressed." Somebody who sees a [*] may wonder what that means, and hopefully find that in the release notes.

Reply:
fwiw I didn't mean for this to hold up the PR, but thanks for taking the suggestion!